nodejs/listeners/actions/block_open_task_list_home.js

```diff
@@ -0,0 +1,19 @@
+const { Task } = require('../../models');
+const { reloadAppHome } = require('../../utilities');
+
+module.exports = (app) => {
+  app.action({ action_id: 'blockOpenTaskCheckboxClicked', type: 'block_actions' }, async ({
+    ack, action, client, body,
+  }) => {
+    await ack();
+    const tasksToUpdate = [];
+    if (action.selected_options.length > 0) {
+      action.selected_options.forEach(async (option) => {
+        const taskID = option.value.split('-')[2];
+        tasksToUpdate.push(taskID);
+      });
+      Task.update({ status: 'CLOSED' }, { where: { id: tasksToUpdate } });
+    }
+    await reloadAppHome(client, body.user.id, body.team.id);
+  });
```

Comment:
As a personal preference for readability, I would replace the `forEach` loop that's populating the `tasksToUpdate` array with a `map`. So instead of:
```
    const tasksToUpdate = [];
    if (action.selected_options.length > 0) {
      action.selected_options.forEach(async (option) => {
        const taskID = option.value.split('-')[2];
        tasksToUpdate.push(taskID);
      });
      Task.update({ status: 'CLOSED' }, { where: { id: tasksToUpdate } });
    }
```

You would have something like this:
```
if (action.selected_options.length > 0) {
    const tasksToUpdate =  action.selected_options.map(async (option) => option.value.split('-')[2])
    Task.update({ status: 'CLOSED' }, { where: { id: tasksToUpdate } });
}

Reply:
I didn't even remember doing the foreach. I usually do map. Good catch!